cmake/TaichiCore.cmake

```diff
@@ -397,12 +397,14 @@ if (NOT WIN32)
         # Linux
         target_link_libraries(${CORE_LIBRARY_NAME} stdc++fs X11)
         target_link_libraries(${CORE_LIBRARY_NAME} -static-libgcc -static-libstdc++)
-        # Avoid glibc dependencies
-        if (TI_WITH_VULKAN)
-            target_link_libraries(${CORE_LIBRARY_NAME} -Wl,--wrap=log2f)
-        else()
-            # Enforce compatibility with manylinux2014
-            target_link_libraries(${CORE_LIBRARY_NAME} -Wl,--wrap=log2f -Wl,--wrap=exp2 -Wl,--wrap=log2 -Wl,--wrap=logf -Wl,--wrap=powf -Wl,--wrap=exp -Wl,--wrap=log -Wl,--wrap=pow)
+        if (TI_ARCH_X64)
```

Comment:
Maybe also add `TI_ARCH_X64` as an option (default ON) on the top? so we don't have to specify at every non-aarch64 build. We have too many flags now..

Reply:
It is the same flag as the backend option. (This does complicate things, hmmm...) I should probably change this to the cmake host arch flags